pilot/cmd/pilot-agent/options/options.go

```diff
@@ -80,6 +80,10 @@ var (
 
 	secretRotationGracePeriodRatioEnv = env.Register("SECRET_GRACE_PERIOD_RATIO", 0.5,
 		"The grace period ratio for the cert rotation, by default 0.5.").Get()
+
+	secretRotationGracePeriodRatioJitterEnv = env.Register("SECRET_GRACE_PERIOD_RATIO_JITTER", .01,
```

Comment:
It's hard to imagine how this could cause problems, but, perhaps this should default to 0 so it doesn't change existing behavior unless folks opt into it? What do ya'll think?

Reply:
I am fine with the default changing